lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarFooter.test.tsx

```diff
@@ -4,63 +4,131 @@
+      quickRender = rtlRender<Props>(NxGlobalSidebarFooter, minimalProps),
+      renderEl = rtlRenderElement<Props>(NxGlobalSidebarFooter, minimalProps);
+
+  it('passes additional classes specified as props to <footer>', function() {
+    const el = renderEl({ className: 'test-class' }),
+        defaultEl = renderEl()!;
 
-  it('renders an <footer> with nx-global-sidebar__footer class and those passed as props', function() {
-    expect(getShallowComponent({ className: 'test-class' }))
-        .toMatchSelector('footer.nx-global-sidebar__footer.test-class');
+    expect(el).toHaveClass('test-class');
+
+    for (const cls of Array.from(defaultEl.classList)) {
+      expect(el).toHaveClass(cls);
+    }
   });
 
   it('passes additional attributes to <footer>', function() {
-    expect(getShallowComponent({ id: 'test-id' })).toHaveProp('id', 'test-id');
-    expect(getShallowComponent({ lang: 'en-ca' })).toHaveProp('lang', 'en-ca');
+    expect(renderEl({ id: 'test-id' })).toHaveAttribute('id', 'test-id');
+    expect(renderEl({ lang: 'en-ca' })).toHaveAttribute('lang', 'en-ca');
   });
 
-  it('renders the Support div with a link with the passed href and text and does not render when empty', function() {
-    const supportEmpty = getShallowComponent().find('div.nx-global-sidebar__support'),
-        support = getShallowComponent({ supportText: 'Support for RSC', supportLink: '#supporturl' })
-            .find('div.nx-global-sidebar__support'),
-        supportLink = support.find(NxTextLink);
+  describe('props', function() {
+    describe('supportText and supportLink', function() {
+      it('renders the Support div with a link with the passed href and text ' +
+      'if supportText and supportLink props are provided', function() {
+        const view = quickRender({ supportText: 'Support for RSC', supportLink: '#supporturl' }),
+            supportLink = view.getByRole('link');
 
-    expect(supportEmpty).not.toExist();
-    expect(support).toContainMatchingElement('span.nx-global-sidebar__support-text');
-    expect(support.find('span.nx-global-sidebar__support-text')).toHaveText('Support for RSC');
-    expect(supportLink).toHaveProp('href', '#supporturl');
-  });
+        expect(view.getByText('Support for RSC')).toBeVisible();
+        expect(supportLink).toHaveTextContent(/^Support for RSC$/);
+        expect(supportLink).toHaveAttribute('href', '#supporturl');
+      });
 
-  it('renders the Release div with a text and version number and does not render when empty', function() {
-    const releaseEmpty = getShallowComponent().find('div.nx-global-sidebar__release'),
-        release = getShallowComponent({ releaseText: 'React Shared Components: 3.1.4' })
-            .find('div.nx-global-sidebar__release');
+      it('renders the Support div with a link with the passed href and jsx ' +
+      'if supportText and supportLink props are provided', function() {
+        const jsx =
+          <>
```

Comment:
Why the fragment?

Reply:
> Why the fragment?

No reason. Something about fragments just make it look more jsx-y

Reply:
> No reason. Something about fragments just make it look more jsx-y

I'm just thinking that without it this assigment would be one line rather than four.

Reply:
> > No reason. Something about fragments just make it look more jsx-y
> 
> I'm just thinking that without it this assigment would be one line rather than four.

Changed here: c134dd4dde59c96fd44c92cd2229bbc9c09e3f56